documentation/disaster-recovery-testing.md

```diff
@@ -0,0 +1,176 @@
+# Disaster Recovery testing
+
+This document covers the Disaster Recovery testing procedure for applications hosted on the Teacher Services AKS clusters.
```

Comment:
How should it be used? As a template and copied to the service repos?

Reply:
My intention was it's left here and all services follow it, as it should cover most/all of our services.

Reply:
OK there may be differences in the repos, but we can revisit that later on